src/portable/status.rs

```diff
@@ -250,9 +250,10 @@ fn normal_status(options: &Status) -> anyhow::Result<()> {
 }
 
 async fn try_get_version(creds: &Credentials) -> anyhow::Result<String> {
-    let mut builder = Builder::uninitialized();
-    builder.credentials(creds)?;
-    let mut conn = Connection::connect(&builder.build()?).await?;
+    let config = Builder::new()
+        .credentials(creds)?
+        .constrained_build()?;
```

Comment:
Seems like `constrained_build()` would raise an error "unsupported constraint builder param" because `credentials` is set.

Reply:
Nice catch. Fixed.